src/api/spec/bootstrap/features/webui/maintained_projects_spec.rb

```diff
@@ -7,11 +7,20 @@
   let(:maintenance_project) { create(:maintenance_project, name: 'maintenance_project', target_project: openSUSE_project) }
 
   describe 'index page' do
-    scenario 'without login' do
-      visit projects_project_maintained_projects_path(project_name: maintenance_project.name)
-      expect(page).to have_text('Maintained Projects')
-      expect(page).not_to have_selector('#new-maintenance-project-modal')
-      expect(page).not_to have_selector('#delete-maintained-project-modal')
+    context 'without login' do
+      before do
+        # The maintenance project factory needs a user logged in, so we fake it just for the creation
+        User.session = admin_user
```

Comment:
this is an anti-pattern. Use admin_user.run_as {}

Reply:
Oh, thanks!